File: app/src/main/java/com/ja/assets/ui/activity/inventory/YesInventoryFragment.java
package com.ja.assets.ui.activity.inventory;

import com.fixed.u8.ui.base.BaseFragment;
import com.ja.assets.R;

public class YesInventoryFragment extends BaseFragment {
    private InventoryActivity inventoryActivity;

    @Override
    protected void setOnCreate() {
        inventoryActivity = (InventoryActivity) getActivity();
    }

    @Override
    protected int getLayoutId() {
        return R.layout.fragment_yes_inventory;
    }

    @Override
    protected void initDataView() {

    }
}

File: app/src/main/java/com/ja/assets/ui/activity/inventory/YesInventoryFragment.kt
package com.ja.assets.ui.activity.inventory

import com.fixed.u8.ui.base.BaseFragment
import com.ja.assets.R

class YesInventoryFragment:BaseFragment(){
    private var inventoryActivity: InventoryActivity? = null
    override fun setOnCreate() {
        inventoryActivity = activity as InventoryActivity
    }

    override fun getLayoutId(): Int = R.layout.fragment_yes_inventory

    override fun initDataView() {

    }
}